mllib-local/src/main/scala/org/apache/spark/ml/linalg/Matrices.scala

```diff
@@ -999,7 +999,7 @@ object Matrices {
             val data = new ArrayBuffer[(Int, Int, Double)]()
             dnMat.foreachActive { (i, j, v) =>
               if (v != 0.0) {
-                data.append((i, j + startCol, v))
+                data.+=((i, j + startCol, v))
```

Comment:
I don't have strong preference, but people might mis-interpret `data += ((i, j + startCol, v))` as adding a tuple of a tuple?